pallets/rmrk-core/src/lib.rs

```diff
@@ -475,6 +475,7 @@ pub mod pallet {
 			new_issuer: <T::Lookup as StaticLookup>::Source,
 		) -> DispatchResult {
 			let sender = ensure_signed(origin.clone())?;
+			ensure!(Self::collections(collection_id).unwrap().issuer == sender, Error::<T>::NoPermission);
```

Comment:
The unwrap here is dangerous. Substrate always expect no panic in the code. We should add a check like this:

```
let collection = Self::collections(collection_id).map_err(Error::<T>::NoCollection)?;
ensure!(collection.issuer == sender, Error::<T>::NoPermission);
```